server/service/src/sync/test/test_data/invoice_line.rs

```diff
@@ -188,7 +189,8 @@ const TRANS_LINE_2: (&str, &str) = (
         "user_8_ID": "",
         "vaccine_vial_monitor_status_ID": "",
         "volume_per_pack": 0,
-        "om_item_variant_id": ""
+        "om_item_variant_id": "",
+        "donor_id": ""
```

Comment:
For good measure... could populate one of these donor_id fields, check that it translates to Some value, like you did for the invoice?